content-services/latest/tutorial/sso/saml.md

```diff
@@ -0,0 +1,351 @@
+| -------- | ----------- |
+| keycloak.auth-server-url |The base URL of the Keycloak, for example `https://keycloak.example.com/auth`|
+| keycloak.realm |The realm name configured in the Keycloak for the Alfresco application, for example `alfresco`|
+| keycloak.resource |The **Client ID** set up in the Desktop Sync for Alfresco Content Services. The client needs to exist underneath the realm set for `keycloak.realm`, for example `desktop-sync`|
+| keycloak.public-client |The adapter will not send credentials for the client to the Keycloak if this is set to true, for example `true`|
+| keycloak.credentials.secret |The secret key for this client if the access type is not set to public.|
+
+## Step 10: Configure Alfresco Process Services
+
+Alfresco Process Services (APS) has two sets of properties that need to be configured to setup SSO. One set synchronizes APS with an LDAP directory and the other set configure with the Keycloak.
+
+1. Configuration for LDAP synchronization can be achieved manually for WAR file deployments using the `activiti-ldap-properties` file or reference an external file for Docker and Kubernetes deployments:
+
+    | Property | Description |
+    | -------- | ----------- |
+    | ldap.authentication.enabled |Sets whether LDAP authentication is enabled. This needs to be `false` as LDAP is only being used for user synchronization, for example `false`|
+    | ldap.authentication.java.naming.provider.url |The URL of the LDAP instance, for example `ldaps://ldap.example.com:636`|
+    | ldap.synchronization.java.naming.security.principal |The user used to access the LDAP directory to perform the synchronization, for example `uid=admin,ou=system`|
+    | ldap.synchronization.java.naming.security.credentials |The password for the user set in `ldap.synchronization.java.naming.security.principal`, for example `secret`|
+    | ldap.synchronization.full.enabled | Sets whether full LDAP synchronization is enabled or not, for example `true`|
+    | ldap.synchronization.full.cronExpression |The cron expression describing how often the full synchronization should run, for example `0 0 0 * * ?`|
+    | ldap.synchronization.differential.enabled |Sets whether differential LDAP synchronization is enabled or not, for example `true`|
+    | ldap.synchronization.differential.cronExpression |The cron expression describing how often the differential synchronization should run, for example `0 0 */4 * * ?`|
+    | ldap.synchronization.userSearchBase |The section of the LDAP directory to restrict user synchronization to, for example `ou=users,dc=alfresco,dc=com`|
+    | ldap.synchronization.groupSearchBase |The section of the LDAP directory to restrict group synchronization to, for example `ou=groups,dc=alfresco,dc=com`|
+
+2. Configuration with the Alfresco Process Services can be achieved manually for WAR file deployments using the `activiti-keycloak.properties` or reference an external file for Docker and Kubernetes deployments:
```

Comment:
```suggestion
2. Configuration with the Alfresco Process Services can be achieved manually for WAR file deployments using the `activiti-identity-service.properties` or reference an external file for Docker and Kubernetes deployments:
```

Same reasoning as per other similar changes, cc: @balsarori in case something changed here and I'm not aware.